graphql_client_codegen/src/query.rs

```diff
@@ -120,21 +120,21 @@ impl<'query, 'schema> QueryContext<'query, 'schema> {
     }
 
     pub(crate) fn response_enum_derives(&self) -> TokenStream {
-        let enum_derives: Vec<_> = self
+        let always_derives = [
+            Ident::new("Eq", Span::call_site()),
+            Ident::new("PartialEq", Span::call_site()),
+        ];
+        let mut enum_derives: BTreeSet<_> = self
```

Comment:
`BTreeSet` is used here so that we don't have duplicate derives, also it would order the derives when iterated.